_modules/conditionals.md

```diff
@@ -6,3 +6,119 @@ short_title: Buttons and Switches
 icon: fa-solid fa-toggle-on
 order: 6
 ---
+
+## What are Conditionals?
+
+In this module, we'll explore **conditionals** which are statements in a program that evaluate to `true` or `false`. You've already seen conditionals when we learned about the **loop condition** in the last module. We'll begin here by exploring the **if statement**, which looks like the following:
+
+```c
+if (buttons_get(1)) {
+    leds_set_color(1, 255, 0, 0);
+}
+```
+
```

Comment:
I think you should point out here that 
```
if (buttons_get(1))
``` 

is the same as 
```
if (buttons_get(1) == true)
```

Reply:
Done